.github/workflows/deploy.yaml

```diff
@@ -66,7 +74,7 @@ jobs:
           context: .
           file: .internal-ci/docker/Dockerfile.frontend
           labels: ${{ steps.docker_meta.outputs.labels }}
-          tags: ${{env.DOCKERHUB_ORG}}/${{env.REPO_NAME}}-frontend:${{ steps.version_output.outputs.version }}
+          tags: ${{env.DOCKERHUB_ORG}}/${{env.REPO_NAME}}-frontend:${{ steps.version_output.outputs.version }}-${{ matrix.versions.prefix }}
```

Comment:
We only build 1 image for the frontend and testnet/mainnet is determined at runtime by passing in env MC_NETWORK=main/test